ci-operator/config/openshift/prometheus-operator/openshift-prometheus-operator-master.yaml

```diff
@@ -1,36 +1,15 @@
-base_images:
-  base:
-    name: "4.15"
-    namespace: ocp
-    tag: base
-  ocp_builder_rhel-8-golang-1.20-openshift-4.15:
-    name: builder
-    namespace: ocp
-    tag: rhel-8-golang-1.20-openshift-4.15
 build_root:
   from_repository: true
 canonical_go_repository: github.com/coreos/prometheus-operator
 images:
 - dockerfile_path: Dockerfile.ocp
   from: base
```

Comment:
shouldn't we remove this line, as in https://github.com/openshift/release/pull/45914/files example? and overall all the from: base as there's no more base defines?

Reply:
yep I realize this after running the rehearsal.